index.html

```diff
@@ -63,6 +63,150 @@ <h3 class="secondaryText">
             <img src="./src/misc/Action.svg" alt="">
         </div>
     </section>
+    <section class="recentWorks centeredGrid">
+        <h2>
+            My Recent <br> Works
+        </h2>
+        <div id="indicator"></div>
+        
+
+        <div class="card workCard">
+            <div class="imagePlaceHolder"></div>
+            <div class="workInfo">
+                <h3  class="cardHeader">Multi-Post Stories <br>
+                    Gain + Glory</h3>
+                <ul class="tagsHandler workTags">
+                    <li>Ruby on Rails</li>
+                    <li>CSS</li>
+                    <li>JavaScript</li>
+                    <li>HTML</li>
+                </ul>
```

Comment:
- [ ] It would be better if you give them some padding to the right and left in order to make it fancier. Like the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468)design 🙏.